init.go

```diff
@@ -1,18 +1,30 @@
 package main
 
 import (
-	"os"
-	"runtime"
-
+	"fmt"
 	"github.com/opencontainers/runc/libcontainer"
+	"github.com/opencontainers/runc/libcontainer/logs"
 	_ "github.com/opencontainers/runc/libcontainer/nsenter"
+	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
+	"os"
+	"runtime"
 )
 
 func init() {
 	if len(os.Args) > 1 && os.Args[1] == "init" {
 		runtime.GOMAXPROCS(1)
 		runtime.LockOSThread()
+
+		err := logs.ConfigureLogging(&logs.LoggingConfiguration{
+			LogPipeFd: os.Getenv("_LIBCONTAINER_LOGPIPE"),
+			LogFormat: "json",
+			IsDebug:   true,
```

Comment:
Why enable debugging globally? Seems a bit extreme. Unfortunately we'd need to pass the logging level by environment variable. :/

Reply:
Done